.ci/integration.sh

```diff
@@ -0,0 +1,50 @@
+. "$CI_SCRIPTS/common.sh"
+
+sudo pip install cpp-coveralls
+
+setup_deps x64
+
+CMAKE_EXTRA_FLAGS="-DTRAVIS_CI_BUILD=ON \
+	-DUSE_GCOV=ON \
+	-DCMAKE_INSTALL_PREFIX= \
+	-DBUSTED_OUTPUT_TYPE=plainTerminal"
+
+PLUGIN_DIR="$TRAVIS_BUILD_DIR/nvim-plugins"
+INSTALL_DIR="$TRAVIS_BUILD_DIR/nvim-install"
+
+$MAKE_CMD CMAKE_EXTRA_FLAGS="$CMAKE_EXTRA_FLAGS" nvim
+NVIM_CMD="$(pwd)/build/bin/nvim"
+export VIMRUNTIME="$(pwd)/runtime"
+#$MAKE_CMD DESTDIR="$INSTALL_DIR" install
+#export VIMRUNTIME="$INSTALL_DIR/share/nvim/runtime"
+#"$INSTALL_DIR"/bin/nvim --version
+
+download() {
+	mkdir -p "$PLUGIN_DIR/$1"
+	git clone -q --recursive "https://github.com/$1" "$PLUGIN_DIR/$1"
```

Comment:
That would break the build if the plugin version we want to use is older than 20 commits (see line below).

With #2340, we could just cache the repos and update them instead cloning from scratch.